src/main/java/com/reckue/post/services/AuthService.java

```diff
@@ -0,0 +1,80 @@
+package com.reckue.post.services;
+
+import java.util.List;
+
+/**
+ * Interface AuthService represents a common base service with CRUD operations for services
+ * with authorization.
+ *
+ * @author Kamila Meshcheryakova
+ * created 14.10.2020
+ */
+public interface AuthService<T> {
```

Comment:
Please explain why we use this service?

Reply:
This service contains special methods related to the token.